README.md

```diff
@@ -127,6 +127,18 @@ Working on project based on boilerplate that used old version of webpack. When I
 * When would you use `document.write()`?
 * What's the difference between feature detection, feature inference, and using the UA string?
 * Explain Ajax in as much detail as possible.
+  AJAX (Asynchronous JavaScript and XML).
+  Web application development technology in which the user interacts with the server in two ways:
+    Synchronous - pausing, the operation of the whole page.
+    Asynchronous - requires callback function, without freezing the application state.
+      There is no assurance that the response to the requests will return to us in the order in which requests are sent.
```

Comment:
Skoro miałem opisać szczegółowo AJAX'a wspomniałem o tym, że zapytanie może być synchroniczne.
http://www.phpmind.com/blog/2010/07/what-is-ajax-synchronous-and-asynchronous/

wg. linka który podałaś 3 parametr przy otwieraniu requesta określa synchroniczność:
httpRequest.open('GET', 'http://www.example.org/some.file', true);
"The optional third parameter sets whether the request is asynchronous. If TRUE (the default), the execution of the JavaScript function will continue while the response of the server has not yet arrived. This is the A in AJAX."